src/InputWithLabel.js

```diff
@@ -0,0 +1,38 @@
+
```

Comment:
You have the right idea here. 
This component is abstract but is everything connected and working properly? I would suggest going over the directions and implementing **exactly** what's described and then after everything is working you can consider further refactoring the component.